cluster/terraform_kubernetes/variables.tf

```diff
@@ -99,7 +99,7 @@ variable "lowpriority_app_replicas" {
 
 variable "kube_state_metrics_version" {
   type    = string
-  default = "2.14.0"
+  default = "2.15.0"
```

Comment:
as this will update for all clusters, can we add to each cluster tfvars.json as we update them? Then remove and update the default when we update the last one (prod).